website/app/styles/pages/application/cover.scss

```diff
@@ -37,9 +37,10 @@
 }
 
 .doc-page-cover__title {
-  @include doc-font-style-h1();
   margin: 0;
   color: var(--doc-color-gray-100);
+
+  @include doc-font-style-h1();
```

Comment:
same here, I'd rather keep it as the first item in the class

Reply:
I'll add comments on these with the issue

Reply:
If requested by the linter it's ok – wanted to make sure I understand the reason